### PR TITLE
Add Plone5 support.

### PR DESCRIPTION
**Pull request by busykoala**
Close https://github.com/4teamwork/izug.organisation/issues/1589

This adds Plone5 support.

### CODE REVIEW

File: ftwbook/graphicblock/testing.py [hunk 1]
```diff
@@ -4,11 +4,12 @@
 from ftw.builder import session
 from ftw.builder import ticking_creator
 from ftw.builder.testing import functional_session_factory
+from ftw.testing import IS_PLONE_5
```
Comment (tinagerber) [inline ftwbook/graphicblock/testing.py]: unused import

File: ftwbook/graphicblock/contents/graphicblock.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class IGraphicBlockSchema(Schema):
         title=_(u'label_file', default=u'PDF file'),
         required=True)
 
-    mode(preview='hidden')
+    omitted('preview')
```
Comment (djowett-ftw) [inline ftwbook/graphicblock/contents/graphicblock.py]: Awkward question no 1) I wonder what the difference is between omitting the field and removing it totally from the schema?

**Comment by djowett-ftw** [comment]
Awkward qu 2) Could we maybe remove `ATContentTypes` from `converter.py`? :-)

**Comment by busykoala** [comment]
> Awkward qu 2) Could we maybe remove `ATContentTypes` from `converter.py`? :-)

If I remember correctly we decided to keep them in a minimal way so it is possible to update to plone 5 (also in a later state).

File: ftwbook/graphicblock/contents/graphicblock.py (hunk 2 above)
Comment (busykoala) [reply ftwbook/graphicblock/contents/graphicblock.py]: I think we need that field so the migration works as expected.

File: ftwbook/graphicblock/testing.py (hunk 1 above)
Comment (busykoala) [reply ftwbook/graphicblock/testing.py]: Thanks, I didn't see that one.

Comment (busykoala) [reply ftwbook/graphicblock/testing.py]: Gets removed in 3067a51

**Comment by djowett-ftw** [comment]
Ok - answer accepted for 1. And 2 will need to be done sometime, but not nec now

**Pull request merged**